content/getting-started/app-guides/chat-actions/index.mdx

```diff
@@ -0,0 +1,70 @@
+---
+weight: 30
+category: "getting-started"
+subcategory: "app-guides"
+title: "Chat Actions"
+tagline: "Create custom Chat Actions for agents."
+desc: "Create custom Chat Actions allowing agents to speed up common tasks."
+---
+
+<p>
+Chat Actions are one of the easiest way to extend the LiveChat Agent Application. Chat Actions are accessible from the contextual menu (represented by <img style="vertical-align: middle" src="./menu-icon.svg" alt="meatballs-menu-icon" />) in the Agent Application. Use them to help agents speed up their daily work by automating common tasks, such as:
+</p>
+
+- opening a link to an external CRM
+- displaying a modal with a form that allows to create a new issue, task, or contact
+- sending a webhook to notify an external service
+- opening an app in the Chat Details section
+
+![Chat Actions in LiveChat Agent App](chat-actions-dropdown-livechat-1.png)
+
+## Building block configuration
+
+Creating an chat action is a three-step process. Come up with a short name that describes the action and decide on the action type.  There are four action types available:
+
+**1. Open a link** provided as the Action target URL in a new tab. See what [parameters](#parameters) we attach to the URL.
+
+**2. Open a modal** with an iframe. The modal will display whatever you provide in the Action target URL.
+
+**3. Send a webook** to the Action target URL. See [what info](#parameters) the webhook will contain.
+
+**4. Open a widget** in the Chat Details section. To use this option, you need to [create an Agent App Widget](/extending-agent-app/agent-app-sdk/#getting-started) first. It's a quick way of opening an app without leaving the chat context.
+
+<div style="max-width: 600px">
+  <img src="chat-action-modal-livechat-example.png" alt="LiveChatChat Action modal" />
+</div>
+
+## Parameters
+
+We attach the following query string parameters to the Action target URL:
+
+<CodeSample>
+
+```
+https://my-target-action-url.com/
+?licence_id=104130623
+&agent_id=smith@example.com
+&time=1604416744108
+&instance_id=0805e283233042b37f460ed8fbf22160  // integration's id
+&source=chats
+&chat_id=PJ0MRSHTDG
+&thread_id=K600PKZON8
+&customer_id=b7eff798-f8df-4364-8059-649c35c9ed0c
+&customer_name=Thomas
+&customer_email=t.anderson@example.com
+&customer_host=<customer_host>
```

Comment:
I'd suggest sticking to sample data, so the formatting is more obvious:
- `<customer_host>` => `customer-isp.remotehost.com`
- `<customer_ip>` => `192.168.1.77`